iroh/src/client.rs

```diff
@@ -60,7 +134,7 @@ where
     }
 
     /// Create a new document.
-    pub async fn create_doc(&self) -> Result<Doc<C>> {
+    pub async fn create(&self) -> Result<Doc<C>> {
```

Comment:
didn't we settle on `new` for creation at some point? cc @b5

Reply:
We did for the CLI, yes. I'm happy to change here too, but wondering: Doesn't `new` here violate Rust naming conventions, as it's not a constructor but a method taking `&self`?

Reply:
I‘d rather be consistent, and use new tbh, what do others think?